addons/gamification/migrations/13.0.1.0/post-migration.py

```diff
@@ -0,0 +1,12 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+_unlink_by_xmlid = [
+    'gamification.mt_badge_granted',
+]
+
+
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    openupgrade.delete_records_safely_by_xml_id(env, _unlink_by_xmlid)
+    openupgrade.load_data(env.cr, 'gamification', 'migrations/13.0.1.0/noupdate_changes.xml')
```

Comment:
detete record translations of email_template_badge_received and simple_report_template templates